app/managers.py

```diff
@@ -0,0 +1,49 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self._cursor = self._connection.cursor()
+
+    def create(self, first_name: str, last_name: str) -> Actor:
+        self._connection.execute(

```

Comment:
Consider adding error handling for database operations to manage exceptions that might occur during the execution of SQL commands. This will make your application more robust.